internal/schema2/memory_2.go

```diff
@@ -27,4 +27,23 @@ type Memory2 struct {
 	// to the VM, allowing it to trim non-zeroed pages from the working set (if supported by
 	// the guest operating system).
 	EnableColdDiscardHint bool `json:"EnableColdDiscardHint,omitempty"`
+
+	// LowMmioGapInMB is the low MMIO region allocated below 4GB.
+	//
+	// TODO: This is pre-release support in schema 2.3. Need to add build number
```

Comment:
@jterry75 how do I resolve these TODOs?

Reply:
I dont think you can. There is no public build that supports this

Reply:
Is there?

Reply:
I think we exposed the MMIO configuration in the public Vb builds.

Reply:
(But not the VPCI configuration)

Reply:
@katiewasnothere - I think you can just update this `TODO` with the `Released in schema 2.3 build ####`